src/api/window/window.ts

```diff
@@ -107,6 +107,10 @@ export interface Area {
     y: number;
 }
 
+interface WindowMovementOptions {
+    moveIndependently: boolean;
+}
+
```

Comment:
The jsdoc for this interface is missing.

Reply:
Added!